### PR TITLE
chore: Add Data binding

### LINKED ISSUE #20: Data  Binding
**Describe the chore**
Currently the app is using the normal findViewById for linking the UI with the methods this will sometimes result in crashes and poor performing apps. It would be really nice if Butter Knife Library by Jake Wharton is used for View Binding

**Would you like to work on the issue?** (Yes/No)
No

### PR DESCRIPTION
**Pull request by Chromicle**
Fixed #20 

Changes: Changed All layouts into data binding layout

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -61,6 +61,7 @@
     <string name="insert_data">Insert Data</string>
     <string name="nothingselected">Nothing selected</string>
     <string name="custompooja">Custom Pooja</string>
+    <string name="amount_inRs">Amount:                                         Rs.</string>
```
Comment (immadisairaj) [inline app/src/main/res/values/strings.xml]: can you change this to `amount_in_rs`

File: app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java [hunk 2]
```diff
@@ -1,138 +1,102 @@
...
-        poojaNotPaid = findViewById(R.id.pooja_not_paid);
-
-        readAll.setOnClickListener(
-                new View.OnClickListener() {
-                    @Override
-                    public void onClick(View view) {
-                        if (donatePaid.isChecked() && donateNotPaid.isChecked()) {
-                            flag = 5;
-                        }
+        readAllBinding.listView.setAdapter(adapter);
```
Comment (immadisairaj) [inline app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java]: indentation

**Comment by immadisairaj** [comment]
@Chromicle also it would be better if you add this in README also

File: app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java (hunk 2 above)
Comment (Chromicle) [reply app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java]: I think everywhere indentation is correct, I run spotless also for checking It didn't give me any errors

**Comment by Chromicle** [comment]
> @Chromicle also it would be better if you add this in README also

I wiil do that now

**Pull request merged**